tests/stdlib/ttypeinfo_std.nim

```diff
@@ -0,0 +1,60 @@
+import typeinfo
+
+type
+  TE = enum
+    blah, blah2
+
+  TestObj = object
+    test, asd: int
+    case test2: TE
+    of blah:
+      help: string
+    else:
+      nil
+
+
+var test = @[0,1,2,3,4]
+var x = toAny(test)
+var y = 78
+x[4] = toAny(y)
+assert x[2].getInt == 2
+
+var test2: tuple[name: string, s: int] = ("test", 56)
+var x2 = toAny(test2)
+var i = 0
+for n, a in fields(x2):
+  case i
+  of 0: assert n == "Field0" and $a.kind == "akString"
```

Comment:
how come this changes from "name" to "Field0" ?

PR's that do moves really shouldn't also do subtle changes, at very least such changes should be described/explained in PR message

Reply:
The tests before are wrong. If you copy the tests to nim playground, you could see that.

Reply:
That's why I say that `when isMainModule` is not tested totally.

https://github.com/nim-lang/Nim/pull/16083#issue-525114167